service/src/main/java/uk/gov/hmcts/reform/fpl/service/ApplicationDocumentsService.java

```diff
@@ -32,6 +84,7 @@ public Map<String, Object> updateApplicationDocuments(List<Element<ApplicationDo
         Map<String, Object> data = new HashMap<>();
 
         data.put("applicationDocuments", updatedDocuments);
```

Comment:
Maybe fine for now but will this not cause duplication? I.e. they'll be added under applicationDocuments and the new fields right?

Reply:
yes, in the case submission state we still use `applicationDocuments` and for a quick fix, we run the migration logic to calculate the new fields.